ci_watson/artifactory_helpers.py

```diff
@@ -88,6 +90,16 @@ def get_bigdata(*args):
     args : tuple of str
         Location of file relative to ``TEST_BIGDATA``.
 
+    repo : str
```

Comment:
Again, don't think `repo` is necessary. See below.

Reply:
Removed in f024a71